src/lambda.d.ts

```diff
@@ -68,7 +69,7 @@ export interface AlphaClientConfig {
   handler: string;
 }
 
-export interface AlphaClient extends AxiosInstance {
+export class LambdaAlphaClient extends Alpha {
```

Comment:
This rename is a breaking change ... is it worth keeping the old name to ease adoption?

Reply:
okay.